### PR TITLE
Script for video addition in ETL

### PR DESCRIPTION
**Pull request by lbucas**
I added a small script which takes art_ontology.json and adds video urls to the according entries.

### CODE REVIEW

**Comment by jschurse** [comment]
Your addition looks good. 
Could you please update the run_all.sh with the according call for your script?

**Comment by jschurse** [comment]
Your videos.csv file would also make a great addition to the repository. Please add it :)

File: scripts/data_manipulation/script_add_videos.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import csv
+import json
+
+videofile_location = "videos.csv"
+ontology_location = "art_ontology.json"
+ontology__output_location = "art_ontology.json"
```
Comment (tilomi) [inline scripts/data_manipulation/script_add_videos.py]: Use as function parameters

File: scripts/data_manipulation/script_add_videos.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+entries_added_count = 0
+for entry in ontology:
+    if entry["id"] in videos and entry['type'] in add_for_types:
+        entries_added_count += 1
+        entry["videos"] = videos[entry["id"]]
+
+print("Added videos for {} entries. Saving the file..".format(entries_added_count))
+
+ontology_out = json.dumps(ontology)
+open(ontology__output_location, 'w').write(ontology_out)
```
Comment (tilomi) [inline scripts/data_manipulation/script_add_videos.py]: Wrap in function e. g. add_youtube_videos(videofile="videos.csv", ontology_location="art_ontology.json", ontology_output_location="art_ontology.json")

Comment (tilomi) [reply scripts/data_manipulation/script_add_videos.py]: If you want to call the script like: "python ./script_add_videos.py" you can add a main function see here https://github.com/hochschule-darmstadt/openartbrowser/blob/master/scripts/upload_to_elasticsearch/elasticsearch_helper.py#L123

**Comment by tilomi** [comment]
I found two types of bugs in the csv you sent me first:
YoutubeMissingIdInCsv
Some videos are missing a id. Remove them or look for the id again please.
Second:
YoutubeTimestampBug
I think one video had this, but the timestamps weren't working for me when using the youtube.com/embed/ endpoint

File: scripts/data_manipulation/script_add_videos.py (hunk 2 above)
Comment (lbucas) [reply scripts/data_manipulation/script_add_videos.py]: Done!

**Comment by lbucas** [comment]
Please review my latest commit. It should resolve the issues. @tilomi Could you add a command to call the script in run_all.sh? I was not sure on where to put it.

**Comment by tilomi** [comment]
I did the changes. Everything works now as expected.
A JSON object with a video looks like this (important for the frontend):
```json
{
    "id": "Q18599452",
    "classes": [
      "Q3305213"
    ],
    "label": "The Lictors Returning to Brutus the Bodies of his Sons. Study",
    "description": "painting by Jacques Louis David",
    "image": "https://upload.wikimedia.org/wikipedia/commons/2/20/The_Lictors_Returning_to_Brutus_the_Bodies_of_his_Sons._Study_%28Jacques_Louis_David%29_-_Nationalmuseum_-_19743.tif",
    "artists": [
      "Q83155"
    ],
    "locations": [
      "Q842858"
    ],
    "genres": [
      ""
    ],
    "movements": [
      ""
    ],
    "inception": "",
    "materials": [
      ""
    ],
    "motifs": [
      ""
    ],
    "country": "",
    "height": "27.5",
    "width": "35",
    "type": "artwork",
    "abstract": "",
    "wikipediaLink": "",
    "absoluteRank": 10,
    "relativeRank": 0.09644813420061217,
    "videos": [
      "https://www.youtube.com/embed/UQiNvDOJi0c"
    ]
  }
```

**Pull request merged**